docs/model-hub/transformers/index.txt

```diff
@@ -0,0 +1,82 @@
+.. _model-hub-transformers:
+
+########################
+ Model Hub Transformers
+########################
+
+.. rubric:: **Overview**
+
+`The Huggingface transformers library
+<https://github.com/huggingface/transformers>`_ is the de-facto library
+for natural language processing models. It provides pretrained weights
```

Comment:
Might want to put the acronym here since you use it later. i.e. "for natural language processing (NLP) models"